Update the rate limit for /users/export/ids from 2,500 -> 250 (#8122)
braze-inc/braze-docs: _includes/rate_limits.md

<!---/users/export/ids-->

{% elsif include.endpoint == "users export ids" %}

> For customers who onboarded with Braze on or after August 22, 2024, This endpoint has a rate limit of 250 requests per minute,as documented in [API rate limits]({{site.baseurl}}/api/api_limits/).

Reviewer: ```suggestion
For customers who onboarded with Braze on or after August 22, 2024, this endpoint has a rate limit of 250 requests per minute, as documented in [API rate limits]({{site.baseurl}}/api/api_limits/).
```

> For customers who onboarded with Braze on or after August 22, 2024, this endpoint has a rate limit of 250 requests per minute, as documented in [API rate limits]({{site.baseurl}}/api/api_limits/).

Reviewer: ```suggestion
If you onboarded with Braze on or after August 22, 2024, this endpoint has a rate limit of 250 requests per minute, as documented in [API rate limits]({{site.baseurl}}/api/api_limits/).
```
